src/components/commons/StatisticPanel.tsx

```diff
@@ -21,6 +21,9 @@ export function StatisticPanel({
   displayId?: string;
   displayStripCustomStyle?: string;
 }) {
+  const displayLength = displayItem.length;
+  const isTabletHorizontal = displayLength <= 2;
```

Comment:
let's remove line 24
```suggestion
  const isHorizontalInTablet = displayItem.length <= 2;
```

Reply:
updated